deps/verifier/src/eventlog/ccel/parser/parsers/ipl.rs

```diff
@@ -0,0 +1,25 @@
+use crate::eventlog::ccel::parser::DescriptionParser;
+use crate::eventlog::ccel::{utils, EventDetails};
+use anyhow::{bail, Error, Result};
+
+pub struct EvIplParser;
+
+impl DescriptionParser for EvIplParser {
+    fn parse_description(&self, data: Vec<u8>) -> Result<EventDetails, Error> {
+        if data.len() < 2 {
+            bail!(
+                "Data is too short: expected at least 2 bytes, got {}",
+                data.len()
+            );
+        }
+        let gce_ipl_prefix = [0x2e, 0x00];
```

Comment:
what's the meaning of this prefix?

Reply:
IPL data payload based on samples and actual GKE result were different -> in case of GKE there is structure in this field (as it's vendor defined) and in for example "CCEL_data_grub_out" it's UTF-8 string as content so to provide backward compatibility I added detecting if IPL is plain utf-8 or structure in UTF-16 prefixed with two bytes